### DISCUSSION TITLE
Upgrade to 1.21.0 version not possible

### DISCUSSION THREAD

**Discussion by fioriniald** [Koenkk/zigbee2mqtt#8237] [General]
Hello,
when I try to upgrade from version 1.19.1 to 1.21.0, after upgrading it doesn't upgrade and remain at version 1.19.1.
The operations I do are as follows: 

```
sudo systemctl stop zigbee2mqtt
cd /opt/zigbee2mqtt
cp -R data data-backup
git checkout HEAD -- npm-shrinkwrap.json
git pull
npm ci
cp -R data-backup/* data
rm -rf data-backup
sudo systemctl start zigbee2mqtt
```

Where is the error?

**Comment by Koenkk** [comment]
After `git checkout HEAD -- npm-shrinkwrap.json` can you do a `git checkout master && git fetch`?

**Reply by fioriniald** [reply]
@Koenkk No, i do not make this operation.
Please tell me if follow is correct:

sudo systemctl stop zigbee2mqtt
cd /opt/zigbee2mqtt
cp -R data data-backup
git checkout HEAD -- npm-shrinkwrap.json
**git checkout master && git fetch**
git pull
npm ci
cp -R data-backup/* data
rm -rf data-backup
sudo systemctl start zigbee2mqtt

Thank you

**Reply by fioriniald** [reply]
Hello,
I tried with the commands above and in home assistant the version does not change and remains on 1.19.1. but i checked that the zigbee2mqtt LOG in opt/zigbee2mqtt/data, shows 1.21.1

Configuration for sensor in HA Core is:
```
  - platform: mqtt
    name: Zigbee2mqtt Version
    state_topic: "zigbee2mqtt/bridge/config"
    value_template: "{{ value_json.version }}"
    icon: mdi:zigbee
```

Thank you
